packages/client/hmi-client/src/components/workflow/ops/compare-datasets/compare-datasets-utils.ts

```diff
@@ -0,0 +1,166 @@
+import { Dataset } from '@/types/Types';
+import { renameFnGenerator } from '@/components/workflow/ops/calibrate-ciemss/calibrate-utils';
+import { DataArray, parsePyCiemssMap, processAndSortSamplesByTimepoint } from '@/services/models/simulation-service';
+import { DATASET_VAR_NAME_PREFIX, getDatasetResultCSV, mergeResults } from '@/services/dataset';
+import { ChartData } from '@/composables/useCharts';
+import { PlotValue } from './compare-datasets-operation';
+
+interface DataResults {
+	results: DataArray[];
+	summaryResults: DataArray[];
+	datasetResults: DataArray[];
+}
+
+export function isSimulationData(dataset: Dataset) {
+	// assume if the dataset has these two files, it's a simulation dataset
+	return (
+		dataset.fileNames?.find((name) => name === 'result.csv') &&
+		dataset.fileNames?.find((name) => name === 'result_summary.csv')
+	);
```

Comment:
I think this is the part where @shawnyama had embedded some telltale metadata if the dataset came from a simulation. I am not totally sure if it is better here, but you may want to if it is worthwhile to sync up so we have one idiomatic way of checking provenance.

Reply:
Actually datasets coming from a simulation always had this `dataset.metadata.simulationId` attribute. I added `dataset.metadata.simulationAttributes` which hold a `modelConfigurationId` and a `interventionPolicyId` but I think its more clear to check for `dataset.metadata.simulationId`

Reply:
I updated to check `dataset.metadata.simulationId` instead. Thanks!